media/img/firefox/new/icon-blocker.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should run these SVGs through https://jakearchibald.github.io/svgomg/ to tidy them up and compress them a bit. It should remove some of the cruft from Sketch.